internal/gengapic/client_init.go

```diff
@@ -53,9 +59,13 @@ func (g *generator) clientOptions(serv *descriptor.ServiceDescriptorProto, servN
 			return errors.E(err, "cannot read default host")
 		}
 
+		if !portRegex.MatchString(host) {
```

Comment:
Would
```go
if !strings.Contains(host, ":") {
  host += ":443"
}
```
work? I'm thinking it doesn't make much sense to define a constant for "443" if it's just going to be used once. Up to you on the regex though :)

Reply:
Yep, I'll go with the simpler approach. Thanks